### PR TITLE
loss module + test loss

### PR DESCRIPTION
**Pull request by QuentinGABOT**

### CODE REVIEW

File: tests/test_datasets.py
Comment (jeremyfix) [inline tests/test_datasets.py:1]: This cannot yet be included because the `PolSFAlos2Dataset` object does not yet have the logic for downloading the data and labels, although these are publicly available.  Therefore I prefer we delete this file for now.

As a sidenote, it is better to make PR on specific issues. Here your PR involves both the loss and the dataset.

**Review by jeremyfix — CHANGES_REQUESTED** [review]

File: tests/test_nn_loss.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Local imports
+import torchcvnn.nn as c_nn
+
+
+def test_complex_mse_loss():
+    loss = c_nn.loss.ComplexMSELoss()
+    y_pred = torch.tensor([1 + 1j, 2 + 2j], dtype=torch.complex64)
+    y_true = torch.tensor([1 + 2j, 2 + 3j], dtype=torch.complex64)
+    output = loss(y_pred, y_true)
+    print(output)
```
Comment (jeremyfix) [inline tests/test_nn_loss.py:36]: Can you assert for the expected value ?

**Review by jeremyfix — CHANGES_REQUESTED** [review]

**Review by jeremyfix — APPROVED** [review]
